### ISSUE TITLE
DnD Support?

### ISSUE THREAD

**Issue by rladstaetter** [gluonhq/substrate#826]
I hope this is the right place to ask:

Using client-maven-plugin, I'm able to compile a little application to a binary (Windows/MacOsX). I'm using dragndrop (dropping a file to the app via mouse) which works if I'm starting the application from intelliJ / with a jvm. 

Unfortunately, it doesn't work when I'm compiling via client-maven-plugin to a native executable. I had to include  ' javafx.scene.input.Dragboard' to the reflectionList parameter, otherwise I get nullpointer exceptions on the console at runtime. 
After adding this in the pom.xml, I don't get any errors, but drag'n drop doesn't work still. 

I tried to find an example in gluon-samples, searched in issues ... but obviously without luck.

So in short, is drag and drop supposed to work currently? 

The repository in question is [javafx-logboard](https://github.com/rladstaetter/javafx-logboard).

**Comment by rladstaetter** [comment]
Sorry for the noise, it was my fault. I could manage to get it to work, the problem sat between keyboard and chair. 👍
